src/components/Layout/Header/NavBar/NavBar.tsx

```diff
@@ -11,10 +13,11 @@ type NavBarProps = {
 
 export const NavBar = ({ isCompact, ...rest }: NavBarProps) => {
   const translate = useTranslate()
+  const { routes: pluginRoutes } = usePlugins()
 
   return (
     <Stack width='full' flex='1 1 0%' {...rest}>
-      {routes
+      {union(routes, pluginRoutes)
```

Comment:
In the future if multiple plugins are displayed on the nav bar, we may need an ordering system to merge routes and plugin routes

Reply:
LGTM, tried without the union and we are just displaying the common routes, so it makes sense that we need it, since previously, no plugin had their own NavBar section.